src/backend/config.ts

```diff
@@ -29,10 +29,10 @@ import {
   getSteamCompatFolder,
   configStore
 } from './constants'
-import { execAsync } from './utils'
 import { execSync } from 'child_process'
 import { logError, logInfo, LogPrefix } from './logger/logger'
 import { dirname, join } from 'path'
+import { execAsync } from './utils/process/process'
```

Comment:
if we create an `index.ts` inside the `process` (and the others) and export things we can import things without this duplication, like `import { execAsync } from './utils/process'`

even more, we could also have an index.ts inside `/utils` that exports everything from every file inside it so we can still do `import { execAsync } from './utils'` like it was before and keep the separation in files for testability/maintainability

Reply:
Sounds good to me. Will work on this.